SPECS/sm.spec

```diff
@@ -168,11 +183,12 @@ fi
 if [ $1 -eq 0 ] ; then
     update-alternatives --remove multipath.conf /etc/multipath.xenserver/multipath.conf
 fi
-exit 0
 
 # XCP-ng
 %systemd_preun linstor-monitor.service
 
+exit 0
```

Comment:
What's the goal of this line here?

Reply:
A failed preun makes a package uninstallable unless you disable scripts with rpm. So I assume they put it as a security in case one of the commands above fails. I think it's best practice to ensure every command returns true, with `|| :`, but I don't know which is best practice.

Reply:
That makes sense, thank you.